agent/grpc-external/services/resource/read.go

```diff
@@ -0,0 +1,54 @@
+package resource
+
+import (
+	"context"
+	"fmt"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+
+	"github.com/hashicorp/consul/internal/resource"
+	"github.com/hashicorp/consul/internal/storage"
+	"github.com/hashicorp/consul/proto-public/pbresource"
+)
+
+func (s *Server) Read(ctx context.Context, req *pbresource.ReadRequest) (*pbresource.ReadResponse, error) {
+
```

Comment:
Super nit-pick 🐜 would you mind nixing this trailing newline? We don't tend to have them at the beginning of function bodies.

Reply:
👍